### ISSUE TITLE
[AGWG] CR transition for ACT-Rules-Format

### ISSUE THREAD

**Issue by nitedog** [w3c/transitions#129]
# Document title, URLs, estimated publication date
Title: Accessibility Conformance Testing (ACT) Rules Format 1.0

Staging draft ready for publication (meets PubRules):
https://www.w3.org/2017/09/ACT/WD-act-rules-format-10-20170912/ 

Planned Publication URI: https://www.w3.org/TR/2019/WD-act-rules-format-1.0-20190416/ 
Latest version URI: https://www.w3.org/TR/act-rules-format/ 

Expected publication date: 16 April 2019

# Abstract
The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test rules. These rules can be evaluated fully-automatically, semi-automatically, and manually. This common format allows any party involved in accessibility testing to document and share their testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing methods, including methods implemented by accessibility test tools.

# Status
This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/. 
This is a Candidate Recommendation of Accessibility Conformance Testing Rules Format 1.0 (ACT Rules Format 1.0) by the Accessibility Guidelines Working Group. This is a call for implementation; the Accessibility Guidelines Working Group requests that implementations be submitted by 28 May 2019. The Working Group targets 28 May 2019 to complete the testing process and produce the implementation report. The Working Group plans to advance past Candidate Recommendation when the Candidate Recommendation Exit Criteria have been met.
The Accessibility Guidelines Working Group primarily seeks feedback in relation to implementation of ACT Rules Format 1.0, but feedback on any aspect of the document is accepted. To comment, file an issue in the W3C ACT TF GitHub repository. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-wcag-act-comments@w3.org (comment archive). Comments are requested by 28 May 2019. In-progress updates to the document may be viewed in the publicly visible editors' draft.
This document was published by the Accessibility Guidelines Working Group as a Candidate Recommendation. This document is intended to become a W3C Recommendation. W3C publishes a Candidate Recommendation to indicate that the document is believed to be stable and to encourage implementation by the developer community. This Candidate Recommendation is expected to advance to Proposed Recommendation no earlier than 28 May 2019.
Please see the Working Group's implementation report.
Publication as a Candidate Recommendation does not imply endorsement by the W3C Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.
This document was produced by a group operating under the W3C Patent Policy. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy. 
This document is governed by the 1 March 2019 W3C Process Document. 

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/w3c-wai-gl/2019JanMar/0371.html

# Changes
Change history: https://w3c.github.io/wcag-act/act-rules-format.html#Change_History

There have been no substantive changes since the Wide Review Working Draft.

Editorially there were several changes to the document since the previous drafts, yet it did not change substantively from a technical perspective. The most significant change was to section 4.4 (previously section 7), which was substantially elaborated. However, that feature of mapping rules to accessibility requirements was there from the beginning. In fact, no features were added or removed in this draft, just their descriptions and formal definitions were adjusted to address comments. Another change was to move the non-normative content of section 4.5 (previously section 8) to the accompanying Working Group Note “Accessibility Conformance Testing Rules: Common Input Aspects”, to facilitate longevity and maintainability of this document intended to become a W3C Recommendation. Most of the other changes were to re-organize the document for better readability and edit sections for better clarity.

# Requirements satisfied
The group believes that the document meets the requirements initially established:
https://w3c.github.io/wcag-act/act-fr-reqs.html

# Dependencies met (or not)
This document does not have dependencies, except RFC 2119 to define keywords. Even though this document was primarily developed for WCAG conformance testing, it is intentionally written to be agnostic to WCAG and any other accessibility standard. For example, this document can be used to create test rules for WAI-ARIA conformance. There are informative references to these standards.

# Wide Review
A Wide Review Working Draft was published on 5 July 2018. No comments were received that required substantive changes (see further explanation under “Changes”). The group followed the checklists for Accessibility, Internationalization, and Privacy/Security, and did not identify any issues relating to these aspects because this document does not specify a new web technology. It defines content that a “rule” should include, and how that content should be written.

# Issues addressed
The group believe to have addressed all comments received on the previous draft:
https://github.com/w3c/wcag-act/labels/For%20CR

# Formal Objections
There were no objections raised for this publication.

# Implementation
A draft implementation report and test plan has been prepared at this location:
	https://w3c.github.io/wcag-act/act-implementations.html
This ACT Rules Format was initially incubated and is continually implemented by the auto-WCAG CG (which is currently changing its name to ACT Rules CG). A list of rules following (the previous draft of) this ACT Rules Format is available already:
	https://auto-wcag.github.io/auto-wcag/pages/rules.html
Many of these existing rules already address these criteria. In fact, most of the feedback on this spec comes from the CG who is actively implementing it on an on-going basis. The CG homepage on GitHub (linked above) also lists automated testing tools and manual testing methodologies that implement these rules published by the CG. The effort work that needs to be made during the CR phase, is to update a set of rules to the latest draft of the ACT Rules Format and document how each of these rules demonstrates successful implementation of the spec features. Therefore, a CR duration of 6 weeks is sufficient to complete the work.

# Patent disclosures
No patent disclosures have been made to date. Patent disclosures for this group are tracked at:
https://www.w3.org/2004/01/pp-impl/35422/status

**Comment by swickr** [comment]
Transition approved.

One suggestion, however: the exit criteria are underspecified; Appendix 4 requires "at least one" whereas our normal expectations are that there be two "independent" implementations.  It's not clear to me whether an "implementation" is a set of rules or an individual rule but in either case wording in the SoTD such as "The Working Group plans to advance past Candidate Recommendation when the implementation report demonstrates [2/3] implementations passing the Candidate Recommendation Exit Criteria."  Feel free to adjust that language to the specific case here.

**Comment by nitedog** [comment]
In agreement with @swickr, changed to:

> The Working Group plans to advance past Candidate Recommendation when at least three independent instances of rules meeting the Candidate Recommendation Exit Criteria have been identified and documented.

**Comment by nitedog** [comment]
Published at https://www.w3.org/TR/2019/CR-act-rules-format-1.0-20190416/
